src/sentry/integrations/slack/utils/channel.py

```diff
@@ -61,14 +60,12 @@ def get_channel_id(
     else:
         timeout = SLACK_DEFAULT_TIMEOUT
 
-    # XXX(meredith): For large accounts that have many, many channels it's
-    # possible for us to timeout while attempting to paginate through to find the channel id
+    # XXX(meredith): For large accounts that have many, many users it's
+    # possible for us to timeout while attempting to paginate through to find the user id
     # This means some users are unable to create/update alert rules. To avoid this, we attempt
     # to find the channel id asynchronously if it takes longer than a certain amount of time,
     # which I have set as the SLACK_DEFAULT_TIMEOUT - arbitrarily - to 10 seconds.
 
-    if features.has("organizations:slack-use-new-lookup", organization):
-        return get_channel_id_with_timeout_new(integration, channel_name, timeout)
     return get_channel_id_with_timeout(integration, channel_name, timeout)
```

Comment:
This looks kinda odd but I renamed the function to not have "new" in it so it is in fact using the newer code.